### ISSUE TITLE
Export to ONNX not working (Stopping without any message)

### ISSUE THREAD

**Issue by joshbigdaddy** [ultralytics/yolov5#4169]
I am using the latest version of yolov5 pulled like 5 mins ago. I am using the yolov5s.pt file to test that export functions properly, but even with this, wich is the simpliest example, i get this error.
## 🐛 Bug

I am trying to do a simple export to test if my onnx module for exporting is well installed, everything seems fine, but doing the export it crashes without saying anything. Find the trace bellow. I have tried preparing a fresh new venv to run everything in a closed environment, but still having the same issues. My whole venv has these packages:

absl-py==0.13.0
attr==0.3.1
attrs==21.2.0
cachetools==4.2.2
certifi==2021.5.30
charset-normalizer==2.0.3
colorama==0.4.4
coremltools==4.1
cycler==0.10.0
flatbuffers==2.0
google-auth==1.33.1
google-auth-oauthlib==0.4.4
grpcio==1.39.0
idna==3.2
kiwisolver==1.3.1
Markdown==3.3.4
matplotlib==3.4.2
mpmath==1.2.1
numpy==1.21.1
oauthlib==3.1.1
onnx==1.9.0
onnx-simplifier==0.3.6
onnxoptimizer==0.2.6
onnxruntime==1.8.1
opencv-python==4.5.3.56
packaging==21.0
pandas==1.3.1
Pillow==8.3.1
protobuf==3.17.3
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyparsing==2.4.7
python-dateutil==2.8.2
pytz==2021.1
PyYAML==5.4.1
requests==2.26.0
requests-oauthlib==1.3.0
rsa==4.7.2
scikit-learn==0.19.2
scipy==1.7.0
seaborn==0.11.1
six==1.16.0
sympy==1.8
tensorboard==2.5.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
thop==0.0.31.post2005241907
torch==1.9.0
torchvision==0.10.0
tqdm==4.61.2
typing-extensions==3.10.0.0
urllib3==1.26.6
Werkzeug==2.0.1

Input:
```
venv_ai\scripts\python.exe export.py --weights yolov5s.pt --img 640 --batch 1
```
Output:
```
export: weights=yolov5s.pt, img_size=[640], batch_size=1, device=cpu, include=['torchscript', 'onnx', 'coreml'], half=False, inplace=False, train=False, optimize=False, dynamic=False, simplify=False, opset=12
YOLOv5  v5.0-318-g0ad6301 torch 1.9.0+cpu CPU

Fusing layers...
C:\Users\Joshbigdaddy\Desktop\MASTER\TFM\code\yolov5\venv_ai\lib\site-packages\torch\nn\functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  ..\c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
Model Summary: 224 layers, 7266973 parameters, 0 gradients

PyTorch: starting from yolov5s.pt (14.8 MB)

TorchScript: starting export with torch 1.9.0+cpu...
C:\Users\Joshbigdaddy\Desktop\MASTER\TFM\code\yolov5\models\yolo.py:57: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if self.grid[i].shape[2:4] != x[i].shape[2:4] or self.onnx_dynamic:
TorchScript: export success, saved as yolov5s.torchscript.pt (29.4 MB)

**ONNX: starting export with onnx 1.9.0...**
```

## Expected behavior
It should export data to ONNX

## Environment

 - OS: [Win10]
 - GPU [RTX 3080]

**Comment by glenn-jocher** [comment]
@joshbigdaddy export works correctly, it's tested every 24 hours and on every commit as part of our CI tests.

It appears you may have environment problems. Please ensure you meet all dependency requirements if you are attempting to run YOLOv5 locally.  If in doubt, create a new virtual Python 3.8 environment, clone the latest repo (**code changes daily**), and `pip install -r requirements.txt` again. We also highly recommend using one of our verified environments below. 


## Requirements

[**Python>=3.6.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.7**](https://pytorch.org/get-started/locally/). To get started:
```bash
$ git clone https://github.com/ultralytics/yolov5
$ cd yolov5
$ pip install -r requirements.txt
```

## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab and Kaggle** notebooks with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>


## Status

![CI CPU testing](https://github.com/ultralytics/yolov5/workflows/CI%20CPU%20testing/badge.svg)

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 training ([train.py](https://github.com/ultralytics/yolov5/blob/master/train.py)), validation ([val.py](https://github.com/ultralytics/yolov5/blob/master/val.py)), inference ([detect.py](https://github.com/ultralytics/yolov5/blob/master/detect.py)) and export ([export.py](https://github.com/ultralytics/yolov5/blob/master/export.py)) on MacOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by xiaodao2049** [comment]
@glenn-jocher I recently came across the same issue when I do exporting with **win10**, but when I using **ubuntu18** everything goes well, Could it be an operating system problem?

### Env
`
cuda 11.1  
`
`
pytorch 1.8.1  
`
`
onnx 1.9.0
`
